capactio/capact
# Increase resource requests and limits for local dev and GKE clusters

**Description**

Changes proposed in this pull request:

- Increase resource requests and limits for local dev and GKE clusters according to short investigation from https://github.com/Project-Voltron/go-voltron/pull/119
- Update Neo4j image
- Add indexes for Public OCH 
- **Additional improvements**: Reorganize repository for cluster configuration

## Testing
1. Run `make dev-cluster` or `make dev-cluster-update`
1. Run `kubectl port-forward -n monitoring svc/monitoring-grafana 3000:80`
1. Observe new resource requests and limits for: Neo4j, Gateway and Public OCH
1. You may also run queries on https://gateway.voltron.local and see if there is a visible change
